x-pack/plugins/fleet/public/plugin.ts

```diff
@@ -243,7 +246,23 @@ export class FleetPlugin implements Plugin<FleetSetup, FleetStart, FleetSetupDep
       Component: LazyCustomLogsAssetsExtension,
     });
 
+    const { capabilities } = core.application;
+    const authz = calculateAuthz({
+      fleet: {
+        // Once we have a split privilege, this should be using fleetv2
+        // all: capabilities.fleetv2.all as boolean,
+        all: capabilities.fleet.all as boolean,
+        setup: false, // browser users will never have setup privileges
```

Comment:
I am kind of confused by this as we currently call the setup from Fleet UI, should we set it to true and change that when we remove that call?